extension/llm/runner/text_prefiller.cpp

```diff
@@ -17,9 +17,11 @@ namespace llm {
 
 TextPrefiller::TextPrefiller(
     TextDecoderRunner* text_decoder_runner,
+    bool use_int32_token,
```

Comment:
oh I meant more like 
```
TextPrefiller::TextPrefiller(
    TextDecoderRunner* text_decoder_runner,
    bool use_kv_cache,
    bool enable_parallel_prefill,
    bool use_int32_token=False)
```
so we don't need to update all callsite...

Reply:
Sorry about misunderstanding. I have updated. Thanks!